dev/run-tests.py

```diff
@@ -649,7 +649,7 @@ def main():
     # if "DOCS" in changed_modules and test_env == "amplab_jenkins":
     #    build_spark_documentation()
 
-    if any(m.should_run_build_tests for m in test_modules) and test_env != "amplab_jenkins":
```

Comment:
Is this related to dropping Python 2.7 support?

Reply:
Oops, no. I was unblock the Jenkins build. I will revert this back.